parent/pom.xml

```diff
@@ -149,17 +124,6 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
-
-      <dependency>
-        <groupId>com.smoketurner.dropwizard</groupId>
```

Comment:
ps I noticed this, and that the [dropwizard-zipkin repository](https://github.com/smoketurner/dropwizard-zipkin) was archived (probably because I was away and @jplock had no other help). If this is still desired, maybe [raise an issue](https://github.com/openzipkin/brave/issues/new/choose) and we can see if there's demand to start a repo again. If not interested please excuse the spam!

Reply:
I’d be happy to unarchive and keep up with releases but I don’t have any bandwidth to review PRs or make major code changes to it.

Reply:
@jplock if you don't mind me ripping out things that may be deprecated and tell me how to cut a release (or mind cutting one if I ping you), let's unarchive it at least until folks get onto brave 6.0 (off deprecated stuff). I can also ask a friend @reta if he can keep an eye out for minor changes. If dropwizard itself changed a lot it might be "beyond my paygrade" wdyt?

Reply:
@codefromthecrypt @jplock  thanks for reaching out, the zipkin stuff and its usage was more of experimental on our end. I did some test drive with it for kafka and (AFAIR) also brave tracing. It turned out that we haven't used these features, and long term solution probably involved opentelemetry and runtime instrumentation, hence we dropped them to reduce burden.

Reply:
all makes sense and thanks for giving things a whirl!